backend/src/main/java/com/google/sps/FindMatchQuery.java

```diff
@@ -47,48 +44,48 @@ public FindMatchQuery(Clock clock, ParticipantDatastore participantDatastore) {
    *     availibility, or null if no match yet
    */
   @Nullable
-  public Match findMatch(Participant newParticipant) {
-    int duration = newParticipant.getDuration();
+  public Match findMatch(Participant firstParticipant) {
```

Comment:
Unless a caller knows the implementation details of this method, `firstParticipant` could be a confusing parameter name. I like the previous `newMatch` or even just `participant`. To make the code in the method easier to read (where there is a second participant to think about) you could add

Participant firstParticipant = participant